src/main/java/lotto/LottoController.java

```diff
@@ -0,0 +1,42 @@
+package lotto;
+
+import lotto.model.*;
+import lotto.model.dto.LottoResultDto;
+import lotto.model.dto.LottoTicketDto;
+import lotto.view.Input;
+import lotto.view.Output;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoController {
+
+    public static void play() {
+        PurchaseAmount purchaseAmount = new PurchaseAmount(Input.getPurchaseAmount());
+        LottoTickets lottoTickets = LottoController.buyLottoTickets(purchaseAmount);
+
+        Output.printPurchaseCount(lottoTickets.getSize());
+        Output.printLottoTickets(lottoTickets.getLottoTickets()
+                .stream()
+                .map(LottoTicketDto::new)
+                .collect(Collectors.toList()));
```

Comment:
아래와 같이 dto변환해주는 로직은 메서드로 분리해준다면 조금 더 가독성이 좋아질 것 같아요. 🙂
```suggestion
        Output.printLottoTickets(parseDto(lottoTickets));
        
      ...
      
      private List<LottoDto> parseDto(LottoTickets lottoTickets) {
          return lottoTickets.getLottoTickets()
                .stream()
                .map(LottoTicketDto::new)
                .collect(Collectors.toList())
          }
```

Reply:
반영하였습니다!